js/chatbot/shared/ChatBot.svelte

```diff
@@ -325,7 +325,10 @@
 
 	.message-wrap > div :not(.avatar-container) :global(img) {
 		border-radius: 13px;
+		margin: var(--size-2);
+		width: 400px;
 		max-width: 30vw;
+		max-height: auto;
```

Comment:
Do you think we should have different sized avatars for mobile / desktop?

Reply:
This actually doesn't affect avatar images, only images uploaded into the chatbot.

Are the avatar images on mobile too large right now?

Reply:
Ah sorry. My mistake. 

No I think avatar images are fine on mobile.